erpnext/quality_management/doctype/customer_feedback/customer_feedback.js

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2018, Frappe and contributors
+// For license information, please see license.txt
+
+frappe.ui.form.on('Customer Feedback', {
+	onload: function(frm){
+		if(!frm.doc.date){
+			frm.set_value("date", frappe.datetime.get_today());
+		}
+	},
+	template: function(frm){	//	Used to fetch the parameters of the selected feedback template
+		if(frm.doc.template){
+			frm.fields_dict.feedback.grid.remove_all();
```

Comment:
You're emptying the child table in both cases. Rather have this outside the `if..else`. Same for the `frm.refresh()`.

Reply:
done